src/views/DisplayError.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div>
```

Comment:
Why a `div` here? Its contents is a single tag, so it's not a must-have in this case.